package.json

```diff
@@ -69,14 +69,8 @@
       }
     },
     "./react-server": {
-      "import": {
-        "types": "./dist/packages/react-server.d.mts",
-        "default": "./dist/packages/react-server.mjs"
-      },
-      "require": {
-        "types": "./dist/packages/react-server.d.ts",
-        "default": "./dist/packages/react-server.cjs"
-      }
+      "react-server": "./dist/packages/react-server.mjs",
```

Comment:
So next would count this as rsc "use server"! As I mentioned, we may need to remove this for now until unstable_postpone is shipped, because for now, this does not do anything special.